Tests/SentryTests/Networking/NSData+Unzip.m

```diff
@@ -0,0 +1,41 @@
+#if __has_include(<zlib.h>)
+#    import <zlib.h>
+#endif
+
+#import "NSData+Unzip.h"
+
+NSData *_Nullable sentry_unzippedData(NSData *data)
+{
+    if (data.length == 0) {
+        return data;
+    }
+
+    z_stream stream;
+    stream.zalloc = Z_NULL;
+    stream.zfree = Z_NULL;
+    stream.avail_in = (uint)data.length;
+    stream.next_in = (Bytef *)data.bytes;
+    stream.total_out = 0;
+    stream.avail_out = 0;
+
+    NSMutableData *decompressed = nil;
+    if (inflateInit2(&stream, 47) == Z_OK) {
```

Comment:
What's this `47`?
Seems to be the `windowBits` field, but it would be better to have a comment explaining why use 47 since we seem to use `(16 + MAX_WBITS)` in `sentry_gzippedWithCompressionLevel`